.github/spot-runner-action/src/ec2.ts

```diff
@@ -51,6 +51,9 @@ export class Ec2Instance {
       this.client = new AWS.EC2({
         credentials: credentials,
         region: this.config.awsRegion,
+        maxRetries: 3,
+        // base 10 seconds for the exponential backoff, up to 3 times
+        retryDelayOptions: {base: 10000}
```

Comment:
Do we also need to update the constructor in L39-L42?

Reply:
Huh this (inherited) code makes no sense, but yeah looks like it :)

Reply:
Ah I see it's for some feature we aren't using, I updated the less important one

Reply:
good catch